share/spice/coderwall/coderwall.js

```diff
@@ -58,8 +52,35 @@
             },
 
             templates: {
-                group: 'icon'
+                group: 'icon',
+                variants: {
+                    iconImage: 'large'
+                },
+                options: {
+                    chompContents: true,
+                    content: DDH.coderwall.content
+                }
             }
         });
+
     };
+
+    Handlebars.registerHelper("parseMD", function(content){
```

Comment:
This is an original handlebars helper. It might be best to swtitch this one out with something that is tried-and-true. The only search result that seemed promising to me was [helper-markdown](https://github.com/helpers/helper-markdown) which has [remarkable](https://github.com/jonschlinkert/remarkable) as a dependency.